api/workloadapi/authorization.proto

```diff
@@ -0,0 +1,92 @@
+syntax = "proto3";
+
+package kmesh.security;
+option go_package="kmesh.net/kmesh/api/workloadapi/security;security";
+
+message Authorization {
+  string name = 1;
+  string namespace = 2;
+
+  // Determine the scope of this RBAC policy.
+  // If set to NAMESPACE, the 'namespace' field value will be used.
+  Scope scope = 3;
+  // The action to take if the request is matched with the rules.
+  // Default is ALLOW if not specified.
+  Action action = 4;
+  // Set of RBAC policy rules each containing its cluases (To, From, When).
+  // If at least one of the rules is matched the policy action will
+  // take place.
+  // Rules are OR-ed.
+  repeated Rule rules = 5;

```

Comment:
which istio version do you referred, it should be ` repeated Group groups = 5;`

Reply:
referred this pr to istio/ztunnel : https://github.com/istio/ztunnel/pull/501

Reply:
That is too old, ambient is iterating fast, we should respect the latest on master

Reply:
This is also the latest code from the ztunnel community.

https://github.com/istio/ztunnel/blob/master/proto/authorization.proto